content/tracing/index.md

```diff
@@ -30,7 +30,7 @@ Install the latest [Datadog Agent](https://app.datadoghq.com/account/settings#ag
 
 To trace applications in Docker containers, you can use the [docker-dd-agent](https://hub.docker.com/r/datadog/docker-dd-agent/) image (tagged version 11.0.5110 or higher) and enable tracing by passing `DD_APM_ENABLED=true` as an environment variable.
 
-For additional information, please reference [the project README on Github](https://github.com/DataDog/docker-dd-agent/blob/master/README.md#tracing--apm)
+For additional information, please reference [the dedicated page](/tracing/docker)
```

Comment:
the wording made me think of kasey casem dedicating a song to some listener on america's top something (some show that died after a long time). "please reference the Docker page"

Reply:
👍